docs/conf.py

```diff
@@ -1,27 +1,38 @@
 # -*- coding: utf-8 -*-
 #
-# alchemtest documentation build configuration file, created by
-# sphinx-quickstart on Fri Mar 24 12:44:43 2017.
+# Configuration file for the Sphinx documentation builder.
 #
-# This file is execfile()d with the current directory set to its
-# containing dir.
-#
-# Note that not all possible configuration values are present in this
-# autogenerated file.
-#
-# All configuration values have a default; values that are commented out
-# serve to show the default.
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/stable/config
+
+# -- Path setup --------------------------------------------------------------
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
 
+# Incase the project was not installed
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
 
-# -- General configuration ------------------------------------------------
+import alchemtest
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'alchemtest'
+copyright = ("2017-2024, The Alchemtest Authors. Project structure based on the "
+             "Computational Molecular Science Python Cookiecutter version 1.1")
```

Comment:
We're grateful to the cookiecutter... but it doesn't need to show up in the copyright line.
```suggestion
copyright = ("2017-2024, The Alchemtest Authors.")
```